File: Entities/DTOs/CarUpdateDto.cs
using Core.Entities;
using System;
using System.Collections.Generic;
using Entities.Concrete;

namespace Entities.DTOs
{
    public class CarUpdateDto : IDto
    {

        public Guid Id { get; set; }
        public Guid BrandId { get; set; }
        public Guid ColorId { get; set; }
        public Guid CarTypeId { get; set; }
        public Guid FuelTypeId { get; set; }
        public Guid GearTypeId { get; set; }
        public Guid InsertedUserId { get; set; }
        public string CarName { get; set; }
        public string Model { get; set; }
        public string Description { get; set; }
        public int MinFindexScore { get; set; }

        public IEnumerable<CarImage> Images { get; set; }

    }
}

File: ui/src/app/models/CarUpdateDto.ts
import { Guid } from "guid-typescript";
import { CarImage } from "./carImage";

export class CarUpdateDto {
  id: Guid;
  brandId: Guid;
  colorId: Guid;
  carTypeId: Guid;
  gearTypeId: Guid;
  fuelTypeId: Guid;
  insertedUserId: Guid;
  carName: string;
  model: string;
  images: CarImage[];
  description: string;
  minFindexScore: number;
  hasChildSeat: boolean;
}
